### PR TITLE
Browser support through MessageChannel.js

### PR DESCRIPTION
**Pull request by cyril-sf**
Browsers that implement `window.postMessage` and not `MessageChannel` are now supported through the `MessageChannel.js` near-polyfill.

### CODE REVIEW

**Comment by cyril-sf** [comment]
@hjdivad good for review.

File: lib/oasis.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import "rsvp" as RSVP;
 
+var URL = this.URL || this.webkitURL;
```
Comment (hjdivad) [inline lib/oasis.js]: You changed this from

``` js
var URL = URL || webkitURL
```

right?

Now it depends on `this` being the global context.  Why not:

``` js
if (typeof URL === 'undefined' && typeof webkitURL !== 'undefined') {
   var URL = webkitURL;
}
```

**Comment by hjdivad** [comment]
@Cyril-sf it looks like you'd already fixed the thing that broke ff.  I still have one minor comment; see what you think.  We can merge either way.

File: lib/oasis.js (hunk 1 above)
Comment (cyril-sf) [inline lib/oasis.js]: I made that change because jshint complained. We would have to add `webkitURL` to `.jshintrc`

Comment (hjdivad) [inline lib/oasis.js]: seems fine no?

**Pull request merged**